src/status_im/extensions/capacities/components.cljs

```diff
@@ -25,26 +28,44 @@
     (js/clearTimeout id))
   (reset! current (js/setTimeout #(on-input-change-text on-change value) delay)))
 
-(defn input [{:keys [keyboard-type style error on-change change-delay placeholder placeholder-text-color selection-color
+(defn- in-range
+  ;; Determines whether a number is in a range
```

Comment:
Should be defined as function doc?

Reply:
I think no need comments at all, the function is pretty simple and self explained